.docs/user-guide/docker-plugins.md

```diff
@@ -278,6 +278,52 @@ Environment Variable | Description | Default | Required
 `GCEPD_TAG` | Only use volumes that are tagged with a label | |
 `GCEPD_ZONE` | GCE Availability Zone | |
 
+## Openstack
+REX-Ray ships with plug-ins for OPenstack as well.
```

Comment:
Hi @sebglon,

Please correct `OPenstack` so that it is `OpenStack`. In fact, please correct all instances of `Openstack` so that they are `OpenStack` as it is the [official casing](https://twitter.com/OpenStack):

![image](https://cloud.githubusercontent.com/assets/101085/26170703/ffcbef78-3b07-11e7-8ce8-d33d13ef476a.png)